src/frontend/utils/dato.ts

```diff
@@ -0,0 +1,28 @@
+import { format, isValid, startOfToday } from 'date-fns';
+
+import type { FeltState } from '@navikt/familie-skjema';
+import { feil, ok } from '@navikt/familie-skjema';
+
+import { datoformat } from './formatter';
+
+export type IsoDatoString = string; // Format YYYY-MM-DD (ISO)
+
+export const dagensDato = startOfToday();
+
+interface FormatterProps {
+    dato?: Date;
+    datoformat: datoformat;
+    defaultString?: string;
+}
+export const formatterDate = ({ dato, datoformat, defaultString = '' }: FormatterProps): string => {
+    return dato && isValid(dato) ? format(dato, datoformat) : defaultString;
+};
+
+export const formatterDateTilIsoString = (dato?: Date): string =>
+    formatterDate({ dato: dato, datoformat: datoformat.ISO_DAG, defaultString: '' });
+
+export const formatterDateTilIsoStringEllerUndefined = (dato?: Date): string | undefined =>
+    dato && isValid(dato) ? format(dato, datoformat.ISO_DAG) : undefined;
```

Comment:
Disse har fått oppdaterte navn i ba-sak. Vil du sørge for at disse også blir oppdatert?

Reply:
Samme med returtypen på de to som er til IsoString
![image](https://github.com/navikt/familie-ks-sak-frontend/assets/25459913/c85bad31-257c-4e03-8da6-09e61efd4b1d)